.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+__Issues are only to be used for bug reports and feature requests.__
+
+Troubleshooting help or questions about installation, configuration, and usage of Janusgraph should be posted in the [JanusGraph users group](https://groups.google.com/forum/#!forum/janusgraph-users).
```

Comment:
```suggestion
Troubleshooting help or questions about installation, configuration, and usage of JanusGraph should be posted in the [JanusGraph users group](https://groups.google.com/forum/#!forum/janusgraph-users).
```

Reply:
@FlorianHockmann I see you proposed a new GitHub feature `Suggested change`. I tried to apply this change (for the first time) it creates a new commit made buy you and makes me as a `co-author`. But the commit isn't signed neither by me nor by you. I don't know if it's a right commit and how it fits a JanusGraph Contribution policy. So, I've amended the change you proposed on my behalf.

I think, before we can use GitHub `suggested change` we should describe the process of such commits here: https://github.com/JanusGraph/janusgraph/blob/master/CONTRIBUTING.md

What do you think?

Reply:
Ah, didn't think about how GitHub would create a commit from such a suggested change. But yes, you're right with amending the commit.
That limits of course the usability of this feature for JanusGraph :-/
The only way I see is that the contributor simply squashes all commits afterwards into a single commit, including the original contributions from the PR.